server-tests/api.tests.js

```diff
@@ -0,0 +1,32 @@
+import { chai } from "meteor/practicalmeteor:chai";
+
+describe("chatter api methods", function () {
+  let user;
+  let room;
+  let meteorUser;
+
+  before(function() {
```

Comment:
inconsistent spacing of `function` and `()` In many of the files. Should have a space.